src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -3905,51 +3905,3 @@ body.tribe-welcome #fs_connect {
 		width: 300px;
 	}
 }
-
```

Comment:
This is great! what I meant was not to include it as a different asset, but leave it as tribe-common-admin and include it via `@import`. (for example, what we're doing [here](https://github.com/the-events-calendar/event-tickets/blob/master/src/resources/postcss/tickets.pcss) or [here](https://github.com/the-events-calendar/event-tickets/blob/master/src/resources/postcss/tickets-commerce.pcss))